lib/rides/pages/search_ride_page.dart

```diff
@@ -126,133 +103,296 @@ class _SearchRideFormState extends State<SearchRideForm> {
     return null;
   }
 
-  Future<void> _onSubmit() async {
-    if (_formKey.currentState!.validate()) {
-      await Navigator.of(context).pushReplacement(
-        MaterialPageRoute<void>(
-          builder: (BuildContext context) => SearchSuggestionPage(
-            _startSuggestion!,
-            _destinationSuggestion!,
-            _selectedDate,
-            _dropdownValue,
+  //todo: get possible Rides from Algorithm
+  Future<void> loadRides() async {
+    if (_startSuggestion == null || _destinationSuggestion == null) return;
+    setState(() => _loading = true);
+    final List<Map<String, dynamic>> data = parseHelper.parseListOfMaps(
+      await SupabaseManager.supabaseClient.from('drives').select('''
+          *,
+          driver:driver_id (
+            *,
+            profile_features (*),
+            reviews_received: reviews!reviews_receiver_id_fkey(*)
+          )
+        ''').eq('start', _startController.text),
```

Comment:
can we maybe add here also that the only the whole day of the datecontroller is selected? or is that another PR and we leave if for now?

Reply:
I wouldn't do this in the databaes query because that would require the results to reload every time you select another day. This would of course mean some more traffic, but I think that is mitigated because we only allow users to create drives up to - what, 3 months? - in the future. And in the end, this is of course replaced by the algorithm

Reply:
yes that's right. I just thought it's kinda weird, that when you only get the rides of the day shown when, whole day is selected and not when a date is selected if i am correct

Reply:
Yes I think the idea has been floated around to also only show drives on the same day without whole day selected, so that the time only makes your search more precise, but I'm not sure if we've discussed that properly

Reply:
yes that seems like a good idea